src/components/LinksSection.jsx

```diff
@@ -0,0 +1,34 @@
+import PropTypes from "prop-types";
+
+const LinksSection = ({ className, linksList }) => {
+  return (
+    <div className={className}>
+      <h3 className="pb-10 text-2xl font-bold uppercase font-heading">Links</h3>
+      <div className="flex flex-col gap-5 justify-center items-center w-full lg:flex-row lg:gap-10">
```

Comment:
collapsing to `flex-col` on small viewports is a good call, I would replace `gap-5 lg:gap-10` with `gap-y-5 gap-x-10` to more semantically reflect what you're going for here